src/storyblok/Button.astro

```diff
@@ -2,11 +2,12 @@
 import { storyblokEditable } from "@storyblok/astro";
 
 const { blok } = Astro.props;
-console.log(blok);
 ---
 
 <div {...storyblokEditable(blok)}>
-  <button>
+  <button
+    class="rounded-full py-2 px-4 bg-yellow-400 font-bold hover:border-solig hover:border-black hover:border-[1px] my-10"
```

Comment:
@schabibi1 just to let you know (no problem at all), border is already border-width: 1px, so border-[1px] is not needed, just `border` is enough. Reference: https://tailwindcss.com/docs/border-width

Reply:
@Dawntraoz Ah! I didn't review styling much and thanks for the fix! Makes sense 😉 I'll push the change from this branch💪

Reply:
Forgot I already deleted a branch 😅 Will push in a new one